help_function/src/util_group_load_test.cpp

```diff
@@ -0,0 +1,213 @@
+// ====------ util_group_load_test.cpp------------ *- C++ -* ----===//

```

Comment:
Can we keep the filename same  as other onedpl test?

Reply:
I think the other similar filenames for sort etc should also change when being migrated to the group_util.hpp file. 
This was changed to util_group_load_test post @danhoeflinger 's review.

Reply:
This shouldn't be done in this one PR.

Reply:
This is not testing oneDPL or oneDPL-like functionality.  It is currently placed in the dpcpp extensions header with the intention to move it to a new header which is targeted at sycl compat.  

This code tests a helper function called from within a kernel, which is a lower level than the rest of oneDPL or oneDPL compatibility headers.

If we must have the name as onedpl test until the code is moved, that is fine, but it is just more that needs to be changed later.

Reply:
Yes noted, will update the file paths when writing linking tests for https://github.com/oneapi-src/SYCLomatic/pull/1784 .